app/src/main/java/com/globant/example/mentorapp/mvp/base/BaseRecyclerViewAdapter.java

```diff
@@ -0,0 +1,16 @@
+package com.globant.example.mentorapp.mvp.base;
+
+/**
+ * Generic class to define basic RecycleView behavior
+ * Created by oscar.amezquita on 21/07/2017.
+ */
+
+public class BaseRecyclerViewAdapter {
```

Comment:
I thought to use it in the repository list when the user selects a repository, open the repository website, but this functionality will be in another PR.